### ISSUE TITLE
PintRun expensive even for nBlocks=10 and nIter=10

### ISSUE THREAD

**Issue by JensHahne** [Parallel-in-Time/time4apint#9]
The construction of the PintRun quickly becomes very expensive, for example with N=10 and K=10. It seems that the calls to simplify() and subs() from sympy are quite expensive. The functions are called for every n,k combination in the PinTRun, resulting in very expensive generations. Even for N=10 and K=10, this can easily lead to 4-5 seconds to create the PintRun.

I don't have a good idea at the moment how to fix this, but with these long runtimes for simulating relatively small block iterations, it is not really reasonable.

**Comment by remi-cssb** [comment]
You are talking about generating the graph to compute the speedup, right ? Maybe write a script somewhere that plot the computation time for different K and N, so we can use it as a base benchmark test to improve the current implementation.

**Comment by tlunet** [comment]
You're talking about the graph generation for speedup computation, right ? Maybe write a script somewhere that compute and plot the computation time for this, so we can use that as a benchmark and try to improve the implementation ...

**Comment by JensHahne** [comment]
Yes, exactly. Good idea, i will add a script and try to improve things.
